### DISCUSSION TITLE
notification for using more than producing?

### DISCUSSION THREAD

**Discussion by cykeler** [netzero-labs/netzero#28] [Q&A]
Hello, 
I was curious if there is a way to set up an automation to let me know when my house is pulling from the grid or powerwall? In the late afternoon when my solar production drops and I am using more than I am producing I either start pulling from the pw3 or the grid. Is there a way to set up a notification for this. I set one up to let me know when my battery drops below 99% so that may do it, but wondering if there is a better way.
thanks!

**Comment by zigam** [comment]
This is not possible using the app currently, you'd have to use Developer Access. You can currently only trigger automations based on Powerwall state of charge or home usage. I might this in the future.  I'm curious what the use case is -- how would you react to the notification?

**Reply by cykeler** [reply]
I would use it to ping me to turn off non necessary loads.. i.e. portable ac unit, fountain pumps, etc. When the solar is producing I have those things going, but once the solar production drops and house starts to pull from battery or grid I would turn those things off. In an ideal world I would automate it all in home assistant, but in this case I usually just manually take action based on checking state throughout the afternoon.

**Reply by zigam** [reply]
Makes sense, thanks!
